modules/tabs-v2/widgets/tabs-v2.php

```diff
@@ -402,12 +468,21 @@ protected function content_template() {
 			var elementUid = view.getIDInt().toString().substr( 0, 3 ); #>
 			<div class="elementor-tabs-wrapper" role="tablist">
 				<# _.each( settings['tabs'], function( item, index ) {
-				var tabCount = index + 1,
-				tabUid = elementUid + tabCount,
+				var tabCount = index + 1;
+				tabUid = elementUid + tabCount;
```

Comment:
Why?

Reply:
@matipojo , it did not work w/o it , i will recheck